docs/api/1.0.67-SNAPSHOT.md

```diff
@@ -0,0 +1,145 @@
+# API Docs - v1.0.67-SNAPSHOT
```

Comment:
Don't push changes to doc. Please remove them...

Reply:
Remove with 55a96f4